internal/provider/common/acl/acl.go

```diff
@@ -89,3 +95,81 @@ func Schema(readOnly bool) map[string]schema.Attribute {
 		},
 	}
 }
+
+func SharedSetToAccessControl(client *govcd.VCDClient, org *govcd.AdminOrg, input []SharedWithModel) ([]*govcdtypes.AccessSetting, []*SharedWithModel, error) {
+	var output []*govcdtypes.AccessSetting
+	var outputModel []*SharedWithModel
+
+	for _, item := range input {
+		var subjectHref string
+		var subjectType string
+		var subjectName string
+		var oModel *SharedWithModel
+
+		if !item.UserID.IsNull() && !item.UserID.IsUnknown() {
+			userID := item.UserID.ValueString()
+			user, err := org.GetUserById(userID, false)
+			if err != nil {
+				return nil, nil, fmt.Errorf("error retrieving user %s: %s", userID, err)
```

Comment:
```suggestion
				return nil, nil, fmt.Errorf("error retrieving user %s: %w", userID, err)
```
Use %w verb to wrap error